sem_covid/entrypoints/notebooks/pycaret_pwdb_all_classes.ipynb

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        train_data = dataset.sample(frac=0.70, random_state=786)
</pre><p>in production, this will have to be done in a balanced manner per class (i.e. for each column separately not once for the entire dataset)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meaningfy-ws/sem-covid/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>